### DISCUSSION TITLE
Sound alert:: New conversation

### DISCUSSION THREAD

**Discussion by ts2050** [LiveHelperChat/livehelperchat#1974] [General]
We have the following flow: a conversation is initiated using the bot, and then in some cases, they are transferred to an attendant. However, the attendant does not receive a sound alert for this new conversation in their queue.

**Comment by remdex** [comment]
Sorry, but it does work here. It's something else you are missing. For an agent 100% do not miss pending chat you can setup https://doc.livehelperchat.com/docs/auto-assignment.

Also if you are having more than 10 pending chats at a time they can increase how many pending chats they see at a time.
